tests/testthat/test-plumber.R

```diff
@@ -150,9 +150,11 @@ test_that("mounts can be read correctly", {
 
   stat <- PlumberStatic$new(".")
   pr$mount("/static", stat)
+  pr$mount("missing-slash", stat)
```

Comment:
It would be good to have two more cases, with trailing slashes.

Reply:
Will do.